.github/workflows/release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Run the build process with Docker
         uses: addnab/docker-run-action@v3
         with:
-          image: emscripten/emsdk:latest
+          image: emscripten/emsdk:3.1.39
```

Comment:
Staying on `latest` isn't just tempting fate: it's already broken in the past month.